From 646352396b3879f156c342846a52230fffaa3e21 Mon Sep 17 00:00:00 2001
From: Manuel Rego Casasnovas <rego@igalia.com>
Date: Mon, 15 May 2017 14:01:09 +0200
Subject: [PATCH] Avoid flakiness on external/wpt/css/css-ui-3/outline-004.html

This patch uses "ahem" instead of "Ahem" to avoid flakiness
on outline-004.html test. And removes it from TestExpectations.

At the same time it adds a new test font-family-case-insensitive.html
using "ahem" to verify the behavior described on the spec:
https://drafts.csswg.org/css-fonts-3/#font-family-casing

It's expected that this new test starts to be flaky on Windows
bots, so it'd should be flagged on TestExpectations once that happens,
and a specific bug should be reported.

BUG=719299

Change-Id: Ic14b7a7d8f0db0c0c9a5501dd91032cd2758549b
---

diff --git a/css/css-ui-3/outline-004.html b/css/css-ui-3/outline-004.html
index c9e8dcb..7c4cc3a 100644
--- a/css/css-ui-3/outline-004.html
+++ b/css/css-ui-3/outline-004.html
@@ -20,7 +20,7 @@
   line-height: 40px;
   font-size: 40px;
   background: red;
-  font-family: ahem;
+  font-family: Ahem;
 }
 </style>
